[2024-05-21T19:12:52.237+0000] {local_task_job_runner.py:120} INFO - ::group::Pre task execution logs
[2024-05-21T19:12:52.249+0000] {taskinstance.py:2073} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: idealista-pipeline.pull_and_save_from_api manual__2024-05-21T19:10:17.300923+00:00 [queued]>
[2024-05-21T19:12:52.254+0000] {taskinstance.py:2073} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: idealista-pipeline.pull_and_save_from_api manual__2024-05-21T19:10:17.300923+00:00 [queued]>
[2024-05-21T19:12:52.254+0000] {taskinstance.py:2303} INFO - Starting attempt 2 of 2
[2024-05-21T19:12:52.264+0000] {taskinstance.py:2327} INFO - Executing <Task(PythonOperator): pull_and_save_from_api> on 2024-05-21 19:10:17.300923+00:00
[2024-05-21T19:12:52.267+0000] {warnings.py:110} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:61: DeprecationWarning: This process (pid=2030) is multi-threaded, use of fork() may lead to deadlocks in the child.
  pid = os.fork()

[2024-05-21T19:12:52.268+0000] {standard_task_runner.py:63} INFO - Started process 2031 to run task
[2024-05-21T19:12:52.268+0000] {standard_task_runner.py:90} INFO - Running: ['***', 'tasks', 'run', 'idealista-pipeline', 'pull_and_save_from_api', 'manual__2024-05-21T19:10:17.300923+00:00', '--job-id', '346', '--raw', '--subdir', 'DAGS_FOLDER/idealista_dag.py', '--cfg-path', '/tmp/tmpy3wwtwmi']
[2024-05-21T19:12:52.269+0000] {standard_task_runner.py:91} INFO - Job 346: Subtask pull_and_save_from_api
[2024-05-21T19:12:52.292+0000] {task_command.py:426} INFO - Running <TaskInstance: idealista-pipeline.pull_and_save_from_api manual__2024-05-21T19:10:17.300923+00:00 [running]> on host cef94752cfd3
[2024-05-21T19:12:52.336+0000] {taskinstance.py:2644} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='shv' AIRFLOW_CTX_DAG_ID='idealista-pipeline' AIRFLOW_CTX_TASK_ID='pull_and_save_from_api' AIRFLOW_CTX_EXECUTION_DATE='2024-05-21T19:10:17.300923+00:00' AIRFLOW_CTX_TRY_NUMBER='2' AIRFLOW_CTX_DAG_RUN_ID='manual__2024-05-21T19:10:17.300923+00:00'
[2024-05-21T19:12:52.337+0000] {taskinstance.py:430} INFO - ::endgroup::
[2024-05-21T19:12:52.474+0000] {pull.py:50} INFO - Token acquired successfully.
[2024-05-21T19:12:52.475+0000] {minio.py:47} INFO - Minio successfully connected.
[2024-05-21T19:12:52.479+0000] {minio.py:126} INFO - Start reading data from object name last_pull
[2024-05-21T19:12:52.497+0000] {minio.py:14} INFO - Last pull occured less than 1 day ago. No pull needed
[2024-05-21T19:12:52.498+0000] {python.py:237} INFO - Done. Returned value was: None
[2024-05-21T19:12:52.498+0000] {taskinstance.py:441} INFO - ::group::Post task execution logs
[2024-05-21T19:12:52.503+0000] {taskinstance.py:1205} INFO - Marking task as SUCCESS. dag_id=idealista-pipeline, task_id=pull_and_save_from_api, execution_date=20240521T191017, start_date=20240521T191252, end_date=20240521T191252
[2024-05-21T19:12:52.521+0000] {local_task_job_runner.py:240} INFO - Task exited with return code 0
[2024-05-21T19:12:52.532+0000] {taskinstance.py:3482} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2024-05-21T19:12:52.533+0000] {local_task_job_runner.py:222} INFO - ::endgroup::
